metrics/examples/grafana/strimzi-kafka.json

```diff
@@ -386,7 +386,7 @@
         }
       ],
       "thresholds": "0,0",
-      "title": "Online Partitions",
+      "title": "Online Partitions Replicas",
```

Comment:
TBH, I get the point why _Online Partitions_ is confusing. But _Online Partitions Replicas_ sounds terrible. Wouldn't _Online Replicas_ be sufficient? Also, is the value really in showing Online replicas? Wouldn't offline replicas be more interesting?

Reply:
We already have "Offline Partitions Count" and now I feel that the name is wrong there as well. My proposal could sound "terrible" (what's a terrible adjective) but it says what it means, but I am fine using just "Replicas".